workers/generic-worker/mocktc/queue.go

```diff
@@ -352,6 +388,13 @@ func (queue *Queue) ListArtifacts(taskId, runId, continuationToken, limit string
 				Name:        name,
 				StorageType: A.StorageType,
 			}
+		case *tcqueue.LinkArtifactRequest:
+			a = tcqueue.Artifact{
+				ContentType: "text/plain", // unused
```

Comment:
I'm not sure this comment is correct. Didn't you just add this field because it was needed? :stuck_out_tongue:

Reply:
That might have been some copy-🍝!